app/helpers/admin_helper.rb

```diff
@@ -68,6 +68,22 @@ def case_link(text, path, badge= nil)
     )
   end
 
+  def percentage_difference(x,y)
+    numerator = (x-y).to_f
```

Comment:
Isn't it `value = (x-y / y) * 100`?

Reply:
https://github.com/brave-intl/publishers/pull/2214/commits/a22a93349b9c221c0d4c5dd10f2aeb2b6c836df0